THIRD-PARTY-NOTICES.txt

```diff
@@ -5,22 +5,35 @@ In the event that we accidentally failed to list a required notice, please bring
 This project uses other open source projects, which are used under the terms of the following license(s).
 --------------------------------------------------
 
-StyleCopAnalyzers - Lightup
+License notice for StyleCopAnalyzers - Lightup
+-----------------------------------------------------
```

Comment:
Feel free to disregard. What if we would align the length of those separators throughout the file (see the line number 6)